src/plugins/network/network.c

```diff
@@ -50,6 +50,9 @@ int elektraNetworkAddrInfo (Key * toCheck)
 	return 0;
 }
 
+#define NETWORK_ATTEMPTS 3
+#define NETWORK_SECONDS_BETWEEN_ATTEMPTS 5
```

Comment:
This is long delay (users might wait for it):

```suggestion
#define NETWORK_SECONDS_BETWEEN_ATTEMPTS 1
```